apps/src/templates/progress/ProgressPill.jsx

```diff
@@ -50,19 +50,8 @@ class ProgressPill extends React.Component {
       return;
     }
 
-    let url = levels[0].url;
-    let params = {};
-
-    if (selectedSectionId) {
-      params['section_id'] = selectedSectionId;
-    }
-
     const userId = queryParams('user_id');
-    if (userId) {
-      params['user_id'] = userId;
-    }
-
-    return (url += stringifyQueryParams(params));
+    return getBubbleUrl(levels[0].url, userId, selectedSectionId);
```

Comment:
Do we want `preserveQueryParams` (the fourth parameter) to be true or false here?

Reply:
I'm letting it be false (since this will create parity with the functionality that existed before this change). I can't think of other parameters we have that we'd want to include in this url right now, and I think explicitly passing these two parameters makes it easier to determine the variations of the url that we might end up with. I figure in the future if we find a use-case for including another parameter we can change it then. I don't feel very strongly about this though and could go either way